autopilot/stim/sound/__init__.py

```diff
@@ -1 +1,14 @@
-from autopilot.stim.sound.sounds import SOUND_LIST
\ No newline at end of file
+"""Module for generating and playing sounds.
+
+This module contains the following files:
+
+    sounds.py : Defines classes for generating sounds
+    jackclient.py : Define the interface to the jack client
+    pyoserver.py : Defines the interface to the pyo server
+
+The use of pyoserver is discouraged in favor of jackclient. This is
+controlled by the pref `AUDIOSERVER`.
+"""
+
+# Import the list of available sounds
+from autopilot.stim.sound.sounds import SOUND_LIST
```

Comment:
Since we now have registries merged, reminding myself to add sounds to them so we can get rid of SOUND LIST (and also enable sound plugins)

Reply:
done in https://github.com/wehr-lab/autopilot/pull/102/commits/e2facb66e444b7929b8090012526a05a97a4b480